tests/tests.c

```diff
@@ -50,9 +50,9 @@
 
 test_pattern_t test_list[NUM_TEST_PATTERNS] = {
     // ena,  cpu, stgs, itrs, errs, description
-    { true,  ONE,    1,    6,    0, "[Address test, walking ones, no cache] "},
-    {false,  ONE,    1,    6,    0, "[Address test, own address in window]  "},
-    { true,  ONE,    2,    6,    0, "[Address test, own address + window]   "},
+    { true,  SEQ,    1,    6,    0, "[Address test, walking ones, no cache] "},
```

Comment:
You need to drop this commit, which reverts tests 0-2 to their state before your HCC speedup PR was merged.